test/fixtures/config-parsing.ts

```diff
@@ -0,0 +1,5 @@
+export const rawConfig = {
```

Comment:
Why not just put this directly in the test file?

Reply:
I expected more fixtures to be added there so I created the file at the beginning, but ended up with just the single value. I'll inline.